ee/tabby-db/src/password_reset.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
+use chrono::{DateTime, Duration, Utc};
+use sqlx::{prelude::FromRow, query};
+use uuid::Uuid;
+
+use crate::DbConn;
+
+#[derive(FromRow)]
+pub struct PasswordResetDAO {
+    pub user_id: i32,
+    pub code: String,
+    pub created_at: DateTime<Utc>,
+}
+
+impl DbConn {
+    pub async fn create_password_reset(&self, id: i32) -> Result<String> {
```

Comment:
```suggestion
    pub async fn create_password_reset(&self, user_id: i32) -> Result<String> {
```

please also update all `id` occurrence actually means `user_id`